src/encord_active/lib/encord/actions.py

```diff
@@ -157,71 +225,74 @@ def create_project(
-                original_labels = original_label_row["data_units"][original_data["data_unit_hash"]]["labels"]
-                initiated_label_row["data_units"][new_label_row["data_hash"]]["labels"] = original_labels
-                initiated_label_row["object_answers"] = original_label_row["object_answers"]
-                initiated_label_row["classification_answers"] = original_label_row["classification_answers"]
-
-                if original_labels != {}:
-                    initiated_label_row = construct_answer_dictionaries(initiated_label_row)
-                    new_project.save_label_row(initiated_label_row["label_hash"], initiated_label_row)
-
-            elif initiated_label_row["data_type"] == DataType.IMG_GROUP.value:
-                object_hashes: set = set()
-                classification_hashes: set = set()
-
-                # Currently img_groups are matched using data_title, it should be fixed after SDK update
-                for data_unit in initiated_label_row["data_units"].values():
-                    for original_data in original_label_row["data_units"].values():
-                        if original_data["data_hash"] == data_unit["data_title"].split(".")[0]:
-                            data_unit["labels"] = original_data["labels"]
-                            for obj in data_unit["labels"].get("objects", []):
-                                object_hashes.add(obj["objectHash"])
-                            for classification in data_unit["labels"].get("classifications", []):
-                                classification_hashes.add(classification["classificationHash"])
+            if progress_callback:
+                progress_callback((counter + 1) / len(all_new_label_rows))
 
-                initiated_label_row["object_answers"] = original_label_row["object_answers"]
-                initiated_label_row["classification_answers"] = original_label_row["classification_answers"]
+        return new_project
 
-                # Remove unused object/classification answers
-                for object_hash in object_hashes:
-                    initiated_label_row["object_answers"].pop(object_hash)
+    def fix_pickle_files(self, embedding_type: EmbeddingType, renaming_map: dict[str, str]):
+        def fix_pickle_file(embedding: LabelEmbedding, renaming_map: dict[str, str]):
+            embedding["label_row"] = renaming_map[embedding["label_row"]]
+            embedding["data_unit"] = renaming_map[embedding["data_unit"]]
+            url_without_extension, extension = embedding["url"].split(".")
+            changed_parts = [renaming_map[x] if x in renaming_map else x for x in url_without_extension.split("/")]
+            embedding["url"] = "/".join(changed_parts) + "." + extension
+            return embedding
+
+        collection = load_collections(embedding_type, self.project_file_structure.embeddings)
+        updated_collection = [fix_pickle_file(up, renaming_map) for up in collection]
```

Comment:
[nitpick] 💅

method could have a more accurate name

```suggestion
        def update_embedding_identifiers(embedding: LabelEmbedding, renaming_map: dict[str, str]):
            embedding["label_row"] = renaming_map[embedding["label_row"]]
            embedding["data_unit"] = renaming_map[embedding["data_unit"]]
            url_without_extension, extension = embedding["url"].split(".")
            changed_parts = [renaming_map[x] if x in renaming_map else x for x in url_without_extension.split("/")]
            embedding["url"] = "/".join(changed_parts) + "." + extension
            return embedding

        collection = load_collections(embedding_type, self.project_file_structure.embeddings)
        updated_collection = [update_embedding_identifiers(up, renaming_map) for up in collection]
```